src/librustc_trans/trans/mir/block.rs

```diff
@@ -30,7 +30,7 @@ impl<'bcx, 'tcx> MirContext<'bcx, 'tcx> {
     pub fn trans_block(&mut self, bb: mir::BasicBlock) {
         debug!("trans_block({:?})", bb);
 
-        let mut bcx = self.bcx(bb);
+        let mut bcx = self.bcx(bb).build();
```

Comment:
It might be better to just return a `BlockAndBuilder` from `self.bcx(BasicBlock)`.